models/UsersModel.js

```diff
@@ -0,0 +1,16 @@
+var mongoose = require('mongoose');
+
+var modelsRegistro = mongoose.Schema({
```

Comment:
please change the name of "modelsRegistro" to "registerModel"

Reply:
Done!